doc/misc/npm-faq.md

```diff
@@ -371,16 +355,12 @@ The npm open source project, The npm Registry, and [the community
 website](https://www.npmjs.com) are maintained and operated by the
 good folks at [npm, Inc.](http://www.npmjs.com)
 
-## I have a question or request not addressed here. Where should I put it?
+## I have a question or request not addressed here!
 
-Post an issue on the github project:
+Please post an issue on GitHub:
 
 * <https://github.com/npm/npm/issues>
 
-## Why does npm hate me?
```

Comment:
Why, after removing or softening the language up above, would you remove this? This is part of the cornerstone of what the project is trying to do, and is not meant to be ironic or more than a tiny bit tongue in cheek. npm, more than most tools, exists to fulfill its users' needs.